src/patternfly/components/Button/button.hbs

```diff
@@ -1,4 +1,4 @@
-<button class="pf-c-button{{#if button--modifier}} {{button--modifier}}{{/if}}"
+<button class="pf-c-button{{#if button--modifier}} {{button--modifier}}{{/if}}" aria-hidden="true"
```

Comment:
This change isn't needed. See the previous comment.

The issue that was flagged for the Button page is the very last button in the [Button States example](https://pf4.patternfly.org/components/Button/examples/):
```
<button class="pf-c-button pf-m-plain" disabled>
  <i class="fas fa-times" aria-hidden="true"></i>
</button>
```
In this example, the `aria-label="Remove"` is missing. You shouldn't need to update the handlebar files to fix this, because it's already defined for the other examples that are similar. Just this one instance was missing it.

Reply:
The last button within the States examples now has `aria-label="Remove"`.